packages/opc-base/src/opc-base/types.ts

```diff
@@ -0,0 +1,36 @@
+import { CreateFeedbackVariable, FeedbackReturn } from "../gql/types";
+import { OpKeyclockAuthProvider } from "../keyclockAuthProvider/keyclockAuthProvider";
+import { ToastOptions } from "../components/opc-toast/types";
+import { Notification } from "../opc-provider/types";
+import { ApolloClient, FetchPolicy, NormalizedCacheObject } from "@apollo/client/core";
+
+export type KeyClockConfig = {
```

Comment:
```suggestion
export type KeycloakConfig = {
```

Reply:
There are a lot of instances of this `keycloak` typo in this file as well as other files. Can you fix that as well?

Reply:
Thats wierd...I did a file search for this typo and didn't catched any. Let me see

Reply:
Fixed. I was thinking its C small letter. Didn't notice the change of `cloak with clock` 😄  also fixed in dev console developer SSI configuration